config/default.schema.json

```diff
@@ -274,6 +274,11 @@
 			"default":"https://schul-cloud.org/etherpad/api/1",
 			"description": "The etherpad api version uri."
 		},
+		"ETHERPAD_OLD_PAD_URI": {
+			"type": "string",
+			"default":"https://etherpad.schul-cloud.org/p",
```

Comment:
validation is missing: please add format and pattern (having no leading slash) as you add it below

Reply:
added